integration_tests/test_employment.py

```diff
@@ -16,7 +16,7 @@ def test_military_gq_employment(tracked_live_populations):
                 pop["household_id"]
                 == data_values.NONINSTITUTIONAL_GROUP_QUARTER_IDS["Military"]
             )
-            & (pop["age"] == data_values.WORKING_AGE)
+            & (pop["age"] >= data_values.WORKING_AGE + 28 / utilities.DAYS_PER_YEAR)
```

Comment:
Use the simulation fixture and the simulation's time-step rather than hard-coding 28 days.